packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -208,10 +211,18 @@ class FlutterErrorDetails extends Diagnosticable {
     this.library = 'Flutter framework',
     this.context,
     this.stackFilter,
+    this.debugCreator,
     this.informationCollector,
     this.silent = false,
   });
 
+  /// Describer to gather important information about [debugCreator].
+  ///
+  /// This function is only called when [debugCreator] is not null.
+  ///
+  /// [WidgetsBinding] is responsible for binding this method.
```

Comment:
Nit: maybe phrase this just:
```
See also:

  * [WidgetsBinding.someMethod], which binds this method.
```
Someone not using the widget layer could completely reasonably bind this to something different and that would be fine.